client/src/rpc_client.rs

```diff
@@ -764,6 +764,18 @@ impl RpcClient {
         self.send(RpcRequest::GetInflationRate, Value::Null)
     }
 
+    pub fn get_inflation_reward(
+        &self,
+        addresses: &[Pubkey],
+        config: Option<RpcEpochConfig>,
```

Comment:
Wdyt about keeping this input `epoch: Option<Epoch>`? Can construct the config object using `self.commitment_config`, like `get_stake_activation()` above

Reply:
Ah, yes. I should have looked closer at this. Thank you.